antora-ui-camel/src/css/vars.css

```diff
@@ -131,9 +132,9 @@
   --nav-panel-height--desktop: calc(var(--nav-height--desktop) - var(--drawer-height));
   --nav-width: calc(270 / var(--rem-base) * 1rem);
   --doc-max-width: calc(720 / var(--rem-base) * 1rem);
-  --doc-max-width--desktop: calc(828 / var(--rem-base) * 1rem);
+  --doc-max-width--desktop: calc(828 * var(--vw-base--desktop) * 1vw);
   --static-max-width: calc(720 / var(--rem-base) * 1rem);
-  --static-max-width--desktop: calc(1140 / var(--rem-base) * 1rem);
+  --static-max-width--desktop: calc(1140 * var(--vw-base--desktop) * 1vw);
```

Comment:
This change in formulas seems arbitrary, what we had matches this logic: we want to have the width in vw. So to do that we divide the number in px by the base rem size, giving us the number of rems in a number of pixels, when we multiply that by 1 rem to get resulting the measure in vw.

You seem to multiply a number by a percentage and then multiply that by a device width varying unit.  Can you explain the logic behind the changed formula?

Reply:
I have calculated the variable --vw-base--desktop using a viewport width of 1300 (which we can change when we decide a standard value). As explained [here](https://stackoverflow.com/questions/28295072/how-can-i-convert-px-to-vw-in-javascript), the variable `--vw-base--desktop` stores the value of 1px in vw. To get the required max-width in vw, I multiplied the number in px by the variable `--vw-base--desktop`. Then I multiplied the number by 1vw to get the resulting measure in vw.